bin/post_process_sce.R

```diff
@@ -247,7 +248,6 @@ processed_sce <- scater::runPCA(
 )
 
 # calculate a UMAP matrix using the PCA results
-try({
   processed_sce <- scater::runUMAP(processed_sce,
     dimred = "PCA"
```

Comment:
Minor formatting annoyance that styler doesn't catch for some reason. (If most arguments are on separate lines, the first argument should be too)
```suggestion
  processed_sce <- scater::runUMAP(
    processed_sce,
    dimred = "PCA"
```